ui/pages/onboarding-flow/metametrics/metametrics.js

```diff
@@ -183,6 +183,10 @@ export default function OnboardingMetametrics() {
                 href={`#${SECURITY_ROUTE}`}
                 target="_blank"
                 rel="noopener noreferrer"
+                onClick={(e) => {
```

Comment:
As discussed in DM, we should have this link in the "Set advanced privacy settings" screen instead of here. This will match the [designs](https://www.figma.com/file/iCOjL3k1BkQqpYHFUVVFUZ/Extension-Onboarding?node-id=3953%3A33668&t=AzvcbsrYIu0OQuMz-0). I know you are working on it, just pointing it out for other reviewers.